### PR TITLE
Chef attribute access re-factor

### PR DESCRIPTION
**Pull request by martinisoft**
Greetings!

This PR changes all instances (I hope) of method based node attribute access due to a bug that cropped up for me in #330 which I have tracked down to the Chef DSL conflicting with an attribute. My hunch is that this has something to do with other DSL enhancement systems such as chef-vault which mix in enhancements to data bag access.

I went with double quotes everywhere (by recommendation of @portertech) and single quotes where applicable, but please look over it to make sure I didn't miss anything.

### CODE REVIEW

**Comment by martinisoft** [comment]
Confirming this PR fixes the issue I mentioned in #330 after testing locally.

File: recipes/_linux.rb [hunk 1]
```diff
@@ -17,43 +17,43 @@
...
-    url "#{node.sensu.yum_repo_url}/#{repo}/el/$basearch/"
+    repo = node["sensu"]["use_unstable_repo"] ? "yum-unstable" : "yum"
+    url "#{node['sensu']['yum_repo_url']}/#{repo}/el/$basearch/"
     action :add
   end
   repo.gpgcheck(false) if repo.respond_to?(:gpgcheck)
 
   yum_package "sensu" do
-    version node.sensu.version
+    version node['sensu']['version']
```
Comment (portertech) [inline recipes/_linux.rb]: Oh snap, tried to sneak these it? :P

**Comment by jhmartin** [comment]
Isn't the standard to surround literals with ' ' instead of " " ?

**Comment by portertech** [comment]
@jhmartin is there a document? I'm fine either way, as long as the use is consistent.

**Comment by jhmartin** [comment]
@portertech https://github.com/bbatsov/ruby-style-guide  which is codified by RuboCop http://batsov.com/rubocop/, which in many cases can auto-correct most of the issues.

File: recipes/_linux.rb (hunk 1 above)
Comment (martinisoft) [inline recipes/_linux.rb]: Oops! Fixed in next commit.

**Comment by martinisoft** [comment]
I have corrected the strings per your note @jhmartin

**Comment by martinisoft** [comment]
If you end up making a rubocop rule I can run that against this branch @portertech

**Comment by martinisoft** [comment]
I've caught up my branch to latest master to make merging possible again. @portertech

**Comment by scalp42** [comment]
Any chance to merge this @portertech  please ? Otherwise @martinisoft gonna be running behind and have to rebase against master all the time.

Thanks in advance!

**Comment by portertech** [comment]
Merged :+1:

**Comment by scalp42** [comment]
Thanks a lot!

**Pull request merged**